modules/phase_field/test/tests/ExpressionBuilderCoupledVectorTest/testCoupledVector.i

```diff
@@ -0,0 +1,162 @@
+# This simulation predicts GB migration of a 2D copper polycrystal with 15 grains
```

Comment:
Could you simplify this input to the absolute minimum required for testing the coupled vector functionality?